src/errors/apiLimitError.ts

```diff
@@ -0,0 +1,7 @@
+/** Error class for the case when API per month limit is exceeded. */
+export default class ApiLimitError extends Error {
+    constructor() {
+    super('You have exceeded 50,000 requests a month. Visit https://ipinfo.io/account to see your API limits.');
```

Comment:
If we ever update the amount from 50k to even 60k, this will become quite confusing, and just adds another place to remember to update out of the million we already have.

Let's make it a bit more generic:

```suggestion
    super('You have exceeded your total request quota this month. Visit https://ipinfo.io/account to see your API limits.');
```